### ISSUE TITLE
Some training settings

### ISSUE THREAD

**Issue by HolmesShuan** [Tangshitao/CENet#1]
Hi, I recently noticed your marvelous work, very interesting! 

However, when I dug into the source codes, I found some unusual settings:
* [ColorAugmentation](https://github.com/Tangshitao/CENet/blob/8781e406938320239c841e22961297e167627503/imagenet/train_imagenet.py#L116)
* [LabelSmoothingLoss](https://github.com/Tangshitao/CENet/blob/8781e406938320239c841e22961297e167627503/imagenet/train_imagenet.py#L36)
* [Dropout](https://github.com/Tangshitao/CENet/blob/8781e406938320239c841e22961297e167627503/imagenet/models/resnet_v1_ce.py#L151)

Could you please kindly report the Top-1 improvements of CENet without bells and whistles? 

In [ECANet](https://arxiv.org/pdf/1910.03151.pdf), they illustrate the real runtime speed in Table 3. Unfortunately, the number is inconsistent with CENet. I did notice they use 2080Ti and 1080Ti in this paper. However, the relative speedup is still different, which makes me confused.

Looking forward to a reply. Thanks in advance!

**Comment by Tangshitao** [comment]
Label smoothing brings about 0.3 top 1 accuracy increase and the other two are useless. As for the speed, I don't know their configuration.

**Comment by HolmesShuan** [comment]
Thanks for your quick reply.
